python-lib/dku_plugin_config_loading.py

```diff
@@ -0,0 +1,164 @@
+from dataiku.customrecipe import *
+import dataiku
+import pandas as pd
+from dataiku import pandasutils as pdu
+from dataiku.customrecipe import get_recipe_config
+from language_dict import SUPPORTED_LANGUAGES_SPACY
+from dku_config import DkuConfig
+
+#All plugin parameters name
+MATCHING_PARAMS       = ['case_sensitivity',
+                         'lemmatization',
+                         'unicode_normalization'
+                        ]
+
+def get_input_datasets(role):
+    return dataiku.Dataset(get_input_names_for_role(role)[0])
+
+def load_settings():
+    
+    text_input        = get_input_datasets('Document dataset').get_dataframe()
+    ontology_input    = get_input_datasets('Ontology dataset').get_dataframe()
+
+    config=get_recipe_config()
+    
+    
+    input_text_cols   = text_input.columns
+    input_onto_cols   = ontology_input.columns
+    
+    dku_config = DkuConfig(
```

Comment:
Out of curiosity, what are local_vars and local_prefix used for?

Reply:
from documentation : "The DkuConfig object also supports local vars. If you want to use a Dku App, the only way to get user settings is to use local vars. A good practice is to prefix these vars by the name of your plugin like that : MY_AWESOME_PLUGIN__param1. That way, there won't be any conflicts if another plugin uses local vars."
so actually..we don't need it :) been removed here  50189a6